banzai/qc/pointing.py

```diff
@@ -61,9 +61,11 @@ def do_stage(self, images):
             elif pointing_warning:
                 self.logger.warning('Pointing offset exceeds threshhold',
                                     extra=self.logging_tags)
-            self.save_qc_results({'PointingSevere': pointing_severe,
-                                  'PointingWarning': pointing_warning,
-                                  'pointing_offset': angular_separation},
+            self.save_qc_results({'pointing.severe.failed': pointing_severe,
+                                  'pointing.severe.threshold': self.SEVERE_THRESHOLD,
+                                  'pointing.warning.failed': pointing_warning,
+                                  'pointing.warning.threshold': self.WARNING_THRESHOLD,
+                                  'pointing.offset': angular_separation},
```

Comment:
I don't really like failed here. Maybe triggered? We should think a bit about what happens if we have tiered thresholds like here with a warning then a severe.

Reply:
I agree, I think "triggered" works better for the warnings. Alternatively, I could take out the distinctions between warning and severe explicitly and just use "pointing.warning" for warning and "pointing.failed" for severe.

Reply:
I like your second suggestion of using point.warning and pointing.failed.

Reply:
Done